src/components/ParserComponent.tsx

```diff
@@ -10,7 +10,7 @@ import Data
   from './Data';
 import * as TimSort from 'timsort';
 import 'bootstrap/dist/css/bootstrap.min.css';
-
+let FileType = '';
```

Comment:
This should be a class property, not a global variable right?

Reply:
No i had to make it a global variable because I could not find a way to access the contents in the file outside of parse to make a mockfile in render() where parse is called and fileReader.result was the only way to access the contents in the file which i assigned to the global variable so i could access it outside of parse